config/config.dist.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use SprykerEco\Shared\UnzerApi\UnzerApiConstants;
+
+$config[UnzerApiConstants::UNZER_PRIVATE_KEY] = '';
```

Comment:
```suggestion
$config[UnzerApiConstants::UNZER_PRIVATE_KEY] = 'YOUR_PRIVATE_KEY_FROM_UNZER';
```

If someone somehow forgot to enter the key, it could be easier to detected this while value is given instead of empty string. What you mean about this idea?